specutils/analysis/width.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+from astropy.stats.funcs import gaussian_sigma_to_fwhm
+from ..spectra import SpectralRegion
+from ..manipulation import extract_region
+from . import centroid
+
+
+__all__ = ['gaussian_sigma_width', 'gaussian_fwhm', 'fwhm']
+
+
+def _computation_wrapper(func, spectrum, region):
+
+    # No region, therefore whole spectrum.
+    if region is None:
+        return func(spectrum)
+
+    # Single region
+    elif isinstance(region, SpectralRegion):
+        return func(spectrum, region=region)
+
+    # List of regions
+    elif isinstance(region, list):
+        return [func(spectrum, region=reg) for reg in region]
+
+
+def gaussian_sigma_width(spectrum, region=None):
+    """
+    Estimate the width of the spectrum using a second-moment analysis, scaled
+    to match the sigma/standard deviation parameter of a standard Gaussian
+    profile. This will be calculated over the regions, if they are specified.
+
+    Parameters
+    ----------
+    spectrum : `~specutils.spectra.spectrum1d.Spectrum1D`
+        The spectrum object over which the width will be calculated.
+
+    region: `~specutils.utils.SpectralRegion` or list of `~specutils.utils.SpectralRegion`
+        Region within the spectrum to calculate the gaussian sigma width. If
+        region is `None`, computation is performed over entire spectrum.
+
+    Returns
+    -------
+    full_width : `~astropy.units.Quantity` or list (based on region input)
```

Comment:
I think this is should be called `sigma` or maybe `sigma_estimate`, right?  That is, it's not a full width (sigma is sort of a "half width", although it's a specific scaling)

Reply:
Yup, another oversight.